### PR TITLE
Update Resource Server sample to ASP.NET Core 2.2.

### PR DESCRIPTION
**Pull request by laura-rodriguez**
Recreates the solution using ASP.NET Core 2.2 framework and template. It follows the recommended best practices such as enforcing https. 
Modifies the README with the http/https updates.

### CODE REVIEW

File: resource-server/README.md [hunk 1]
```diff
@@ -21,22 +21,38 @@ A typical resource-server requires a frontend and a backend application, so you
...
+> **NOTE:** This sample is using ASP.NET Core 2.2 which enforces HTTPS. This is a recommended practice for web applications. Check out [Enforce HTTPS in ASP.NET Core] for more details.
 
-**front-end:**
+#### Visual Studio
 
-If you want to use one of our front-end samples, open a new terminal window and run the [front-end sample project of your choice](Prerequisites).  Once the front-end sample is running, you can navigate to http://localhost:8080 in your browser and log in to the front-end application.  Once logged in, you can navigate to the "Messages" page to see the interaction with the resource server.
+If run this project in Visual Studio it will start the resource server on ports 5000 for HTTP and 44314 for HTTPS. You can change this configuration in the `launchSettings.json`.  You can browse to `https://localhost:44314/api/messages` to ensure it has started. If you get a 401 HTTP error, it indicates that the resource server is up. You will need to pass an access token to access the resource, which will be done by the front-end below.
```
Comment (OkGoDoIt) [inline resource-server/README.md]: If _you_ run this... (typo)

File: resource-server/okta-aspnetcore-webapi-example/Controllers/MessageController.cs [hunk 2]
```diff
@@ -24,12 +25,12 @@ public JsonResult Get()
...
                 ?.Value;
 
-            return Json(new
+            return new JsonResult(new
             {
                 messages = new dynamic[]
                 {
-                    new { Date = DateTime.Now, Text = "I am a Robot." },
-                    new { Date = DateTime.Now, Text = "Hello, world!" },
+                new { Date = DateTime.Now, Text = "I am a Robot." },
```
Comment (OkGoDoIt) [inline resource-server/okta-aspnetcore-webapi-example/Controllers/MessageController.cs]: Indentation (although not entirely sure in this instance of dynamic object initializer)

File: resource-server/okta-aspnetcore-webapi-example/Properties/launchSettings.json [hunk 3]
```diff
@@ -3,10 +3,11 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:8000/",
-      "sslPort": 0
+      "applicationUrl": "http://localhost:5000",
+      "sslPort": 44314
```
Comment (OkGoDoIt) [inline resource-server/okta-aspnetcore-webapi-example/Properties/launchSettings.json]: In https://github.com/okta/samples-aspnetcore/pull/23/files#diff-6f9d3b74cd5f3e0e8e6ff04b0b1abe89R38 it says the https port is 5001.  Does that need to be updated here or is this something different?

Comment (laura-rodriguez) [reply resource-server/okta-aspnetcore-webapi-example/Properties/launchSettings.json]: It's 5001 for dotnet CLI (Kestrel). Unfortunately, we can't use the same range of ports for IIS, that's why for that case we are using 44314.

**Pull request merged**